pkg/manager/impl/util/shared.go

```diff
@@ -233,3 +234,22 @@ func GetTaskExecutionModel(
 	}
 	return &taskExecutionModel, nil
 }
+
+// GetMatchableResource gets matchable resource for resourceType and project - domain combination.
+// Returns an error if such a resource is not found.
```

Comment:
Can you add that this returns nil if nothing is found but it won't error?

Reply:
Done. misread the function